voluptuous/humanize.py

```diff
@@ -0,0 +1,39 @@
+from voluptuous import Invalid, MultipleInvalid
+from voluptuous.error import Error
+
+
+MAX_VALIDATION_ERROR_ITEM_LENGTH = 500
+
+
+def _nested_getitem(data, path):
+    for item_name in path:
+        try:
```

Comment:
not using EAFP here will make the code more complex since we're dealing with lists/tuples/other indexables in addition to dicts, but I'll give it a shot.